CHANGELOG.md

```diff
@@ -41,6 +41,8 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 - [#2450](https://github.com/thanos-io/thanos/pull/2450) Store: regex-set optimization for `label=~"a|b|c"` matchers.
 - [#2475](https://github.com/thanos-io/thanos/pull/2475) Store: remove incorrect optimizations for queries with `=~".*"` and `!=~".*"` matchers.
 
+- [#2194](https://github.com/thanos-io/thanos/pull/2194) Updated to golang v1.14.2
```

Comment:
This needs to be in unreleased, can update your PR if you don't mind (:

Reply:
Actually will wait for you, no rush (: 

I am adding requirements for our tools to be on go 1.14, that's why want to finalize this PR before next release ! (:

Reply:
I updated the changelog :)